src/specs/App.spec.js

```diff
@@ -0,0 +1,14 @@
+import { create } from 'react-test-renderer';
+import App from '../App';
+
+jest.mock('../components/Container', () => 'Container');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
jest.mock("../components/Container", () => 'Container');
```